### ISSUE TITLE
Show warning only instead of stopping when @cover references a wrong target

### ISSUE THREAD

**Issue by HappyRay** [sebastianbergmann/phpunit#1829]
Currently when a test method references a non existing class e.g.

    /**
     * @covers Box\TestScribe\ClassNameUtil
     */
    public function testGetNormalizedFullyQualifiedClassNameStartWithBackSlash()

The class has been moved to Utils namespace
Box\TestScribe\Utils\ClassNameUtil

test coverage process will stop with the following error:

php ~/phar/phpunit.phar --coverage-text /Users/ryang/github/TestScribe/tests/Unit/Utils/ClassNameUtilGenTest.php
PHPUnit 4.8.4 by Sebastian Bergmann and contributors.

Runtime:	PHP 5.4.37 with Xdebug 2.2.3
Configuration:	/Users/ryang/github/TestScribe/phpunit.xml

.Trying to @cover or @use not existing method "Box\TestScribe\ClassNameUtil::getNormalizedFullyQualifiedClassName".

----

Due to a limitation of Intellij's inability to update these references when refactoring code, it's frustrating and tedious to have to find and fix these errors one at a time.

It would be nice for the test coverage tool to ignore these wrong directives and give warnings about all of them at once.

**Comment by sebastianbergmann** [comment]
I think that erroring out is the "right thing" to do. You can use a static analysis tool such as https://github.com/dunkelfrosch/phpcoverfish to find wrong `@covers` annotations.

If IntelliJ/PhpStorm is not able to automatically update these annotations then this is something that @mikhailvink and his team should know (and do something :-) about.

**Comment by HappyRay** [comment]
Oh, I wasn't aware that this functionality was covered by a separate project. Good to know. Thanks.

I didn't know phpcoverfish. I took a quick look. It seems to be helpful. With that tool, the problem I am facing may become less of an issue. Thanks. 

Previously, my colleagues were so frustrated with this issue and strongly suggested that I stop using this annotation and stop generating this annotation from my open source test generation automation tool [testScribe](https://github.com/box/TestScribe). This is a bigger issue for my company Box.com because we run many tests in parallel. Many integration tests are slow. We didn't have an effective way to find all such @cover errors at once. I will try phpcoverfish when I get a chance or recommend it to others.

FYI I reported the intellij issue. It is being worked on. [Here](https://youtrack.jetbrains.com/issue/WI-28486) is the ticket.
